CHANGES.rst

```diff
@@ -1,5 +1,6 @@
 2.0.0 (Unreleased)
 ==================
+- [BREAKING] Removed make_result method from View and Query classes.
```

Comment:
I think it might be good to direct people to the appropriate methods to use here in case anyone was using `make_result`.